### PR TITLE
Bug 1136902 - Added test file and method for utils sanitize method

### PR DESCRIPTION
**Pull request by sr-murthy**
Bug 1136902 - Added test file and method for utils sanitize method.

### CODE REVIEW

File: firefox_puppeteer/tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness import FirefoxTestCase
+
+
+class TestUtils(FirefoxTestCase):
```
Comment (whimboo) [inline firefox_puppeteer/tests/test_utils.py:8]: I would add a different class for each helper method. Otherwise it will become too big once more helper methods are contained in that file.

File: firefox_puppeteer/tests/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          root.containerOpen = true;
+          for (let i = 0; i < root.childCount; i++) {
+            urls.push(root.getChild(i).uri)
+          }
+          root.containerOpen = false;
+
+          return urls;
+        """)
+
+    def test_sanitize(self):
```
Comment (whimboo) [inline firefox_puppeteer/tests/test_utils.py]: It would be better to have different test methods for each individual data type.

File: firefox_puppeteer/tests/test_utils.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+        # Call the utils sanitize().
+        self.utils.sanitize(data_type={"history": True, "cookies": True})
```
Comment (whimboo) [inline firefox_puppeteer/tests/test_utils.py]: We should ensure that the cookie is set. Same for the history which should contain x items.

File: firefox_puppeteer/tests/test_utils.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+        # Check that the browser history has been cleared
+        self.assertEqual(len(self.get_all_urls_in_history()), 0)
```
Comment (whimboo) [inline firefox_puppeteer/tests/test_utils.py:65]: It would be better to to leave out the length and compare the arrays directly. That way we could see which history items are still present.

File: firefox_ui_tests/resources/cookie_single.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (whimboo) [inline firefox_ui_tests/resources/cookie_single.html:1]: Please put this file into the `cookies` sub folder so we keep those files separate by module.

File: firefox_puppeteer/tests/test_utils.py (hunk 1 above)
Comment (sr-murthy) [inline firefox_puppeteer/tests/test_utils.py:8]: I'm not sure I understand.  Do you mean `get_all_urls_in_history`?  That is the only helper method I see, but you want me to put that in a class of its own?  Where should the class live, or is there an example of something similar?

File: firefox_puppeteer/tests/test_utils.py (hunk 2 above)
Comment (sr-murthy) [inline firefox_puppeteer/tests/test_utils.py]: You mean a different sanitize method for different types of resource, like pages, cookies, etc.

File: firefox_puppeteer/tests/test_utils.py (hunk 1 above)
Comment (whimboo) [inline firefox_puppeteer/tests/test_utils.py:8]: Gna, partly scratch my last comment. It should not have been `helper method` but methods in utils.js. Means each of those methods in utils.js should better get its own class due to complexity. Alone for sanitize we need a couple of test methods in the class.

File: firefox_puppeteer/tests/test_utils.py (hunk 2 above)
Comment (whimboo) [inline firefox_puppeteer/tests/test_utils.py]: Exactly. Otherwise it will be very hard to follow once we have tests for all types.

File: firefox_puppeteer/tests/test_utils.py (hunk 1 above)
Comment (sr-murthy) [inline firefox_puppeteer/tests/test_utils.py:8]: Sure, no problem.
